src/master/util.ts

```diff
@@ -182,19 +182,15 @@ export async function loadChannel(
                         avatar: channel.client.user.displayAvatarURL()
                     })
                     .then(async webhook => {
-                        const messages = (channelData as TextChannelData).messages
+                        let messages = (channelData as TextChannelData).messages
                             .filter(m => m.content.length > 0)
                             .reverse();
-                        let i = 0;
+                        messages = messages.slice(messages.length - options.maxMessagesPerChannel);
```

Comment:
This would be better, I think:
```js
const messages = (channelData as TextChannelData).messages
                            .filter(m => m.content.length > 0)
                            .reverse()
							.slice(messages.length - options.maxMessagesPerChannel);
```

Reply:
Can't use `messages.length` inside `message`'s declaration

Reply:
Yes, you're right... My bad 🙄